static/scripts/calendar.js

```diff
@@ -49,6 +49,26 @@ const getCalendarLanguage = (langAttribute) => {
 	}
 };
 
+// Add data-testid attributes to navigation buttons in calendar page as the calendar module
+// that is used (https://fullcalendar.io) does not support configuration of data-testids for buttons in used version
+const addDatatestidsToCalendarNavigation = () => {
```

Comment:
```suggestion
const addDataTestIdsToCalendarNavigation = () => {
```
... to better readability.

Reply:
Done